arrayTransformations.mjs

```diff
@@ -7,15 +7,16 @@
 // --------------------------------------------------------------------------
 
 //requiring it locally:
-// let buildArray = require('./dataToArray.js').buildArray;
+import { buildArray } from './dataToArray.mjs';
+import * as R from 'ramda';
 
 /**
-  * Set a size of the array and the array will be expanded or shrunk to fix that size. To expand it will just loop the array:
-  * Does the same thing as adjustArrayLength but does it differently
-  * @param {number} number - the desired length of adjusted array
-  * @param {array} array - the array to modify
-  * @example console.log(resizeArray(5, [0,1,2])) // [0, 1, 2, 0, 1]
-*/
+ * Set a size of the array and the array will be expanded or shrunk to fix that size. To expand it will just loop the array:
+ * Does the same thing as adjustArrayLength but does it differently
+ * @param {number} number - the desired length of adjusted array
+ * @param {array} array - the array to modify
+ * @example console.log(resizeArray(5, [0,1,2])) // [0, 1, 2, 0, 1]
+ */
```

Comment:
Sorry, I use prettier (https://prettier.io/) for auto formatting on save. So this kind of change is caused by it.